Sources/SectionScrubber.swift

```diff
@@ -33,7 +33,12 @@ public class SectionScrubber: UIView {
         guard let collectionView = self.collectionView else { return 0 }
         guard let window = collectionView.window else { return 0 }
 
-        if collectionView.superview?.isKindOfClass(NSClassFromString("UICollectionViewControllerWrapperView")!) != nil {
+        /*
+         We check against the `UICollectionViewControllerWrapperView`, because this indicates we're working with
+         a collection view that is inside a collection view controller. When that is the case, we have to deal with its 
+         superview instead of with it directly, otherwise we have a offsetting problem.
+         */
+        if collectionView.superview?.isKindOfClass(NSClassFromString(String.init(format: "U%@ectionViewCont%@w", "IColl", "rollerWrapperVie"))!) != nil {
```

Comment:
> Is there another way to ask for the adjusted container origin? Could we use a delegate instead? Moving the responsibility to the person that implements the control?

Then we're just pushing the mess down the line. Instead of us doing this _hacky_ code, we're asking users to do it for us. 💭 I don't see a way around this unless we enforce that this can't be used with `UICollectionViewController`.